src/gather/tests/test_commands.py

```diff
@@ -44,61 +52,78 @@ def fooish():
 @NON_CONFLICTING_COMMANDS.register(name='weird_name')
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name1():
+    """One of several commands registering for same name"""
     pass
 
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name2():
+    """One of several commands registering for same name"""
     pass
 
 @CONFLICTING_COMMANDS.register(name='weird_name')
 def weird_name3():
+    """One of several commands registering for same name"""
     pass
 
 class CollectorTest(unittest.TestCase):
 
+    """Tests for collecting plugins"""
+
     def test_collecting(self):
+        """Collecting gives only the registered plugins for given collector"""
         collected = NICE_COMMANDS.collect()
         self.assertIn('main1', collected)
         self.assertIs(collected['main1'], main1)
         self.assertNotIn('baz', collected)
 
     def test_non_collision(self):
+        """Collecting with same name for different collectors does not collide"""
         nice = NICE_COMMANDS.collect()
         evil = EVIL_COMMANDS.collect()
         self.assertIs(nice['weird_name'], main2)
         self.assertIs(nice['bar'], main3)
         self.assertIs(evil['weird_name'], main3)
 
     def test_cross_module_collection(self):
+        """Collection works when plugins are registered in a different module"""
         collected = _helper.WEIRD_COMMANDS.collect()
         self.assertIn('weird_function', collected)
 
     def test_transform(self):
+        """Collecting transformd plugins applies transform on collection"""
         collected = TRANSFORM_COMMANDS.collect()
         self.assertIn('fooish', collected)
         res = collected.pop('fooish')
         self.assertIs(res.original, fooish)
         self.assertEquals(res.extra, 5)
 
     def test_one_of_strategy(self):
+        """:code:`one_of` strategy returns one of the registered plugin for name"""
```

Comment:
_The_ `one_of` strategy... the registered ~plugin~ plugins

Is there a way to indicate that `name` was a parameter?